src/main/scala/org/hammerlab/guacamole/commands/jointcaller/AlleleAtLocus.scala

```diff
@@ -104,34 +104,38 @@ object AlleleAtLocus {
       val subsequenceCounts =
         ReadSubsequence.nextAlts(pileup.elements)
           .filter(subsequence => !onlyStandardBases || subsequence.sequenceIsAllStandardBases)
-          .groupBy(x => (x.endLocus, x.sequence))
-          .map(pair => (pair._2.head -> pair._2.length))
+          .groupBy(x => (x.endLocus, x.sequence()))
```

Comment:
if a method's declaration has `()` then IntelliJ warns against not calling with `()`; I don't know the significance otomh but it feels reasonable to be consistent with the declaration. In particular, you can't go the other direction.